src/helpers/measureTitleHelper.ts

```diff
@@ -48,6 +47,15 @@ function getDerivedMeasureTitleBase(masterMeasure: VisualizationObject.IMeasure)
     return get<string>(masterMeasure, ['measure', 'alias'], masterMeasureTitle);
 }
 
+function findOverTimeComparisonType(definition: IMeasureDefinitionType) {
```

Comment:
I suggest renaming `definition` to `definitionType` will be helpful :)

Reply:
Fixed.